activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/service/ModelingServiceAutoConfiguration.java

```diff
@@ -73,13 +73,15 @@ public ModelService modelService(ModelRepository modelRepository,
                                      ModelContentService modelContentService,
                                      ModelExtensionsService modelExtensionsService,
                                      JsonConverter<Model> jsonConverter,
-                                     ProcessModelContentConverter processModelContentConverter) {
+                                     ProcessModelContentConverter processModelContentConverter,
+                                     Set<ModelUpdateListener> modelUpdateListeners) {
```

Comment:
No, there is no ModelUpdateListener by default (at least at the community level).

I am not completely sure that it will fail on startup as this very same `Registry pattern` has been implemented in the same way with the `ContentUpdateListener` and it doesn't seem to be failing. Anyway, I think it is a good practice to add the annotation to help developers to understand the code.

Reply:
Ok, It sounds good then. I had some trouble with that in the past, but it looks like it has been fixed in the latest versions of Spring.